udata/tests/api/test_user_api.py

```diff
@@ -60,130 +60,57 @@ def test_unfollow(self):
         self.assertEqual(Follow.objects.following(user).count(), 0)
         self.assertEqual(Follow.objects.followers(user).count(), 0)
 
-    def test_suggest_users_api_first_name(self):
```

Comment:
Could you add user suggest tests back since we're not removing the suggest in this PR? :pray: